incubator/external-dns/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: Deploys external-dns into a kubernetes cluster
+name: external-dns
+version: 0.0.1
```

Comment:
We can enrich the yaml similar to https://github.com/kubernetes/charts/blob/master/stable/mediawiki/Chart.yaml. This allows us to have both the chart version and external-dns's version in here.